rspotify-http/src/reqwest.rs

```diff
@@ -85,23 +59,26 @@ impl ReqwestClient {
         log::info!("Making request {:?}", request);
         let response = request.send().await?;
 
+        // Making sure that the status code is OK
         if response.status().is_success() {
             response.text().await.map_err(Into::into)
         } else {
-            Err(HttpError::from_response(response).await)
+            Err(ReqwestError::StatusCode(response))
```

Comment:
It seems you don't show how to construct the `ReqwestError::Client()`? When will then client return this error type?

Reply:
It's just the same as `reqwest::Error`, so whenever that happens. For example when making the request or when converting the reply into a String.